mew-ssl.el

```diff
@@ -125,8 +128,13 @@ insert no extra text.")
 	    (insert mew-prog-ssl-arg))
 	(insert (format "[%d]\n" localport))
 	(insert (format "accept=%s:%d\n" mew-ssl-localhost localport))
-	(insert (format "connect=%s:%d\n" server remoteport))
-	(if tls (insert (format "protocol=%s\n" tls)))
+	(if mew-ssl-proxy-server
+	    (insert
+	     (format "connect=%s:%s\nprotocol=connect\nprotocolHost=%s:%d\n"
+		     mew-ssl-proxy-server mew-ssl-proxy-port
```

Comment:
Use the functions defined in mew-config.el if you'd like to support mew-config-alist.
For reference, see https://github.com/kazu-yamamoto/Mew/pull/162

Reply:
thank you, fixed them.